modules/ROOT/pages/try-scope-concept.adoc

```diff
@@ -132,6 +133,15 @@ In all these cases the transaction created at Flow level will be rolled back, an
 * Always Begin (`ALWAYS_BEGIN`): Since the transaction that runs within the `try` Scope is different from the one that runs in the flow, the transaction from the scope is committed and the message is sent.
 * Begin or Join (`BEGIN_OR_JOIN`): Now, the `jms:publish` operation is tied to the commit of the flow transaction, even if the `try` Scope has finished its execution. Considering that the error raised in the flow is not handled (with an `on-error-continue`) and the transaction is rolled back, this includes rolling back the `jms:publish` operation and not sending the message. 
 
+=== Local or XA Transactions
+
+In addition to Transactional Action, you can configure the Transaction Type to Local (single Resource) or XA Transaction. See xref:transaction-management.adoc[Transaction Management] for more information about transactions types. 
+
+When setting the Transaction Type you must consider that a Transaction cannot change it's type in the middle of its execution. So, for the different Transactional Action you get different behaviors:
+
+* `INDIFFERENT`: The Transaction used is the one already created (if there is one). This means that setting the Transactional Type makes no difference for this Transactional Action. 
+* `ALWAYS_BEGIN`: The Transaction created is of the type set in the Transaction Type configuration. Remember that `ALWAYS_BEGIN` is an invalid configuration when already running within a Transaction.  
```

Comment:
As far as I know is not invalid. It's valid (because it's going to work) but it will commit the current transaction in order to open the new one.

Reply:
No, this is not a valid configuration.